CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+
+If you are a new contributor see: [Steps to Contribute](#Steps-to-Contribute)
+
+If you have a trivial fix or improvement, go ahead and create a pull request, addressing (with @...) a suitable maintainer of this repository (see CODEOWNERS of the repository you want to contribute to) in the description of the pull request.
+
+If you plan to do something more involved, first discuss your ideas on our mailing list gardenlinux@googlegroups.com. This will avoid unnecessary work and surely give you and us a good deal of inspiration.
+
+## Steps to Contribute
+
+Should you wish to work on an issue, please claim it first by commenting on the GitHub issue that you want to work on it. This is to prevent duplicated efforts from contributors on the same issue.
+
+If you have questions about one of the issues, with or without the tag, please comment on them and one of the maintainers will clarify it.
+
+We kindly ask you to follow the [Pull Request Checklist](#Pull-Request-Checklist) to ensure reviews can happen accordingly.
+
+## Contributing Code
+
+You are welcome to contribute code to Garden Linux in order to fix a bug or to implement a new feature.
+
+The following rules govern code contributions:
+
+1. Contributions must be licensed under the MIT License.
+2. You need to sign the Developer Certificate of Origin.
+
+## Contributing Documentation
+
+You are welcome to contribute documentation to Garden Linux.
+
+The following rules govern documentation contributions:
+
+1. Contributions must be licensed under the Creative Commons Attribution 4.0 International License.
+2. You need to sign the Developer Certificate of Origin.
+
+## Developer Certificate of Origin
+
+Due to legal reasons, contributors will be asked to accept a Developer Certificate of Origin (DCO) before they submit the first pull request to this projects, this happens in an automated fashion during the submission process. We use the standard DCO text of the Linux Foundation.
+
+## Pull Request Checklist
+
+1. Branch from the master branch and, if needed, rebase to the current master branch before submitting your pull request. If it doesn’t merge cleanly with master you may be asked to rebase your changes.
+
+2. Commits should be as small as possible, while ensuring that each commit is correct independently (i.e., each commit should compile and pass tests).
+
+3. Test your changes as thoroughly as possible before your commit them. Preferably, automate your test by unit / integration tests. If tested manually, provide information about the test scope in the PR description.
+
+4. Create Work In Progress [WIP] pull requests only if you need a clarification or an explicit review before you can continue your work item.
+
+5. If your patch is not getting reviewed or you need a specific person to review it, you can @-reply a reviewer asking for a review in the pull request or a comment, or you can ask for a review on our [mailing list].
+
+8. Post review:
```

Comment:
8 follows 5, should be renamed to 6 (or will 6 and 7 show up later)?

Reply:
fixed the numbering and the link